static/css/theme-activestate.css

```diff
@@ -3,217 +3,324 @@
+	font-family: 'Roboto', sans-serif;
+	text-rendering: optimizeLegibility;
+	color: #5e5e5e;
+	font-weight: 300;
+	letter-spacing: 0;
+	text-align: left !important;
+	text-transform: capitalize;
+	line-height: 1.2;
 }
 
-h3 {
-    padding-top: 3rem;
+h1, h2, h3 {
+ 	padding-top: 2.5rem !important;
 }
 
 h4 {
-    font-weight: bold;
-    padding-top: 1rem;
+	font-weight: bold;
+	padding-top: 1rem;
 }
 
-b, strong, label, th  {
-    font-weight: bold;
+#shortcuts > h3 {
+	color: #FFFFFF !important;
+}
+
+p {
+	font-size: 1rem !important;
+ 	margin-top: 0 !important;
+}
+
+b, a, strong, label, th {
+	font-weight: 500;
 }
 
 .progress ul {
-    padding: 10px 15px 10px 30px; 
+	padding: 10px 15px 10px 30px;
 }
 
 #TableOfContents {
-    font-size: 13px !important;
-    max-height: 85vh;
-    overflow: auto;
-    padding: 0 !important;
+	font-size: 13px !important;
+	max-height: 85vh;
+	overflow: auto;
+	padding: 0 !important;
 }
 
 #TableOfContents li {
-    list-style-type: disc;
-    padding: 0;
+	list-style-type: disc;
+	padding: 4px 0;
 }
 
 #TableOfContents > ul > li > a {
-    font-weight: bold;
+	font-weight: bold;
+	margin: 0;
+	padding: 0;
 }
 
+#header-wrapper {
+	border-top: 8px solid;
+	border-bottom: 0 !important;
+	padding: 1rem;
+}
+
+#leaving-organizations {
+	padding-top: 0 !important;
+}
+
+#related-information {
+	padding-top: 0 !important;
+}
 
 #sidebar #shortcuts h3 {
-    font-family: 'Roboto', sans-serif;
-    color: white ;
-    margin-top:1rem;
-    padding-left: 1rem;
+	font-family: 'Roboto', sans-serif;
+	color: #FFFFFF;
+	margin-top:1rem;
+	padding-left: 1rem;
+}
+
+.searchbox {
+	border: 0;
+	border-bottom: 1px solid;
+	border-radius: 0 !important;
+}
+
+.searchbox input {
+	background: var(--MENU-HEADER-BG-color);
+    border: 0 !important;
+    border-radius: 0 !important;
+}
+
+#body #breadcrumbs a{
+	color: var(--MENU-HEADER-BORDER-color);
+	text-decoration-color: var(--MENU-HEADER-BORDER-color);
+}
+
+#body #breadcrumbs a:hover{
+	color: var(--MENU-HEADER-BORDER-color);
+	text-decoration: underline;
+	text-decoration-color: var(--MENU-HEADER-BORDER-color) !important;
+}
+
+#top-github-link a {
+	color: var(--MENU-HEADER-BORDER-color);
+}
+
+mark {
+    background: var(--MAIN-BG-color) !important;
+    color: #333333;
+}
+
+.nav {
+	color: var(--MENU-HEADER-BORDER-color);
+}
+
+.nav:hover {
+	color: var(--MENU-HEADER-BORDER-color);
 }
 
 .children-li p {
-    font-size: small;
-    font-style: normal !important;
-    margin-top: -24px !important;
-    padding-bottom: 24px !important;
+	font-size: small;
+	font-style: normal !important;
+	margin-top: -24px !important;
+	padding-bottom: 24px !important;
 }
 
 li {
-    padding: 0 0 1.5rem 0;
+	padding: 0 0 1.25rem 0;
 }
 
 ol{
-    padding-top: 0;
+	padding-top: 0;
+	margin-top: 0;
+}
+
+ol ul {
+	margin-top: 1rem;
+}
+
+.children li a {
+	padding-bottom: 1rem;
+}
+
+.hljs {
+	display: block;
+	overflow-x: auto;
+	padding: 0.5em;
+	line-height: 1.3em;
+	color: #2D52BD;
```

Comment:
Looks like we re-use this value a lot and it's also defined as a variable at the top. Can you replace all instances of it with the variable reference?